biobakery_all.snakefile

```diff
@@ -2,13 +2,12 @@
 
 import os
 
-configfile: "config.yaml"
+configfile: "config_template.yaml"
```

Comment:
This should stay `config.yaml` - the idea is that you would copy this template to the folder that you're running the workflow in, and point to that. Take a look at `/home/vklepacc/software/kevin_scratch/test_workflow` as an example.